Makefile

```diff
@@ -5,8 +5,11 @@ OVMF   := ovmf-x64/OVMF_CODE-pure-efi.fd ovmf-x64/OVMF_VARS-pure-efi.fd
 
 all: $(DRIVER) $(GA_ISO) $(OVMF)
 
+test:
+	cd windows-gaming-driver && cargo test --release --locked
```

Comment:
Please add debug tests as well. `debug_assert`s should be verified as soon as possible.